tests/vision/docs/test_plots.py

```diff
@@ -0,0 +1,15 @@
+from runpy import run_path
+import glob
+
+import torch
+
+
+def test_plots_on_gpu():
+    """If there is GPU available running all the docs plot files. Only makes sure the plots don't crash, and not \
+    testing any other display or functionality."""
+    if torch.cuda.is_available() or True:
+        plots_files = glob.glob('../docs/source/**/plot_*.py', recursive=True)
```

Comment:
maybe better to work with Pathlib or os.path to support all OS

Reply:
how do you know glob doesn't support all OS?

Reply:
glob is fine. but the path you specified as parameter - better use os.path

Reply:
I don't really understand why it's better, but updated